unattended.sh

```diff
@@ -347,11 +347,16 @@ function set_required_prerequisite() {
 			subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
 			;;
 
-		centos-release* | centos-linux-release* | centos-stream-release*)
+		centos-release-8.3* | centos-linux-release* | centos-stream-release*)
 			BASE_PACKAGES=(dnf-plugins-core epel-release)
 			$PKG_MGR config-manager --set-enabled powertools
 			;;
 
+		centos-release-8.2*)
```

Comment:
```suggestion
		centos-release-8.[1-2]*)
```
It can be rare, but as there are still people with installation images of 8.2, it also has for 8.1 .. if we extend to 8.1, we have already avoided this issue. :smile: